explorer/explorer.go

```diff
@@ -254,6 +247,9 @@ type explorerUI struct {
 	// page that should be accessible during DB synchronization.
 	displaySyncStatusPage atomic.Value
 	politeiaAPIURL        string
+
+	invsMtx sync.RWMutex
+	invs    *types.MempoolInfo
```

Comment:
Well, I approved but Travis did not.  The Mempool type and it's mutex are unused.

Also, I generally don't like the pattern of a named mutex for a certain field.  I see no issue with embedding a mutex in the type you are locking access on.  Either way it's the user's responsibility to use the lock, but with it in the data type, it's clearer what it's for.  I don't push back though.